pages/blog/[blogId].tsx

```diff
@@ -0,0 +1,69 @@
+import { useRouter } from 'next/router';
+
+import Grid from '@mui/material/Grid';
+import Container from '@mui/material/Container';
+
+import HeroDetails from '../../components/hero/HeroDetails';
+import TextColumn from '../../components/common/text-column/TextColumn';
+import ImagesColumn from '../../components/common/images-column/ImagesColumn';
+import RelatedPosts from '../../components/blog/RelatedPosts';
+import TableOfContent from '../../components/common/table-of-content/TableOfContent';
+
+import { blogData } from '../../mocks/blog-data';
+
+import { socialShareButtons } from '../../utils/socialShareButtons';
+
+const BlogDetailPage = () => {
+  const router = useRouter();
+  const blogId = router.query.blogId;
+
+  const data = blogData.find(item => item.id === Number(blogId));
+
+  const pageLink = `https://5f9731df.next-varbintechcom.pages.dev${router.asPath}`;
+
+  return (
+    <>
+      {data ? (
+        <HeroDetails
+          title={data.blogTitle}
+          projectTags={data.blogTags}
+          projectSocialIcons={socialShareButtons(pageLink)}
+          postAuthor={data.blogAuthor}
+          postDate={data.blogDate}
+          postAuthorPhoto={data.blogAuthorPhoto}
+          postBgImage={data.blogImage.src}
+        />
+      ) : null}
+
+      {data ? (
```

Comment:
It seems we can omit the second check and put the markup on the first one.

Reply:
'+